setup.py

```diff
@@ -30,7 +30,7 @@ def run_tests(self):
 
 setup(
     name='tagcompare',
-    version='0.1.1',
+    version='0.2.0-alpha2',
```

Comment:
You might want to take a look at g-box's setup.py for a way to avoid manually updating this version number all the time.